content/id/docs/concepts/architecture/cloud-controller.md

```diff
@@ -0,0 +1,238 @@
+
+Pada model baru ini, Kubelet menyiapkan sebuah node tanpa informasi spesifik _cloud_. Namun, Kubelet menambahkan sebuah Taint pada node yang baru dibuat yang menjadikan node tersebut tidak dapat dijadwalkan (sehingga tidak ada Pod yang dapat dijadwalkan ke node tersebut) hingga CCM menyiapkan node tersebut dengan informasi spesifik _cloud_. Setelah itu, Kubelet menghapus Taint tersebut.
+
+## Mekanisme _Plugin_
+
+CCM menggunakan _interface_ Go untuk memungkinkan implementasi dari _cloud_ apapun untuk ditambahkan. Secara spesifik, CCM menggunakan CloudProvider Interface yang didefinisikan [di sini](https://github.com/kubernetes/cloud-provider/blob/9b77dc1c384685cb732b3025ed5689dd597a5971/cloud.go#L42-L62)
+
+Implementasi dari empat kontroler-kontroler yang disorot di atas, dan beberapa kerangka kerja, bersama dengan CloudProvider Interface, akan tetap berada pada kode inti Kubernetes. Implementasi spesifik penyedia layanan _cloud_ akan dibuat di luar kode inti dan menggunakan CloudProvider Interface yang didefinisikan di kode inti.
+
+Untuk informasi lebih lanjut mengenai pengembangan _plugin_, lihat [Mengembangkan Cloud Controller Manager](/docs/tasks/administer-cluster/developing-cloud-controller-manager/).
+
+## Otorisasi
+
+Bagian ini memerinci akses yang dibutuhkan oleh CCM terhadap berbagai objek API untuk melakukan tugas-tugasnya.
+
+### Akses untuk Node Controller
+
+Node Controller hanya berinteraksi dengan objek-objek Node. Node Controller membutuhkan akses penuh untuk operasi `get`, `list`, `create`, `update`, `patch`, `watch`, dan `delete` terhadap objek-objek Node.
+
+v1/Node:
+
+* Get
+* List
+* Create
+* Update
+* Patch
+* Watch
+* Delete
+
+### Akses untuk Route Controller
+
+Route Controller memantau pembuatan objek Node dan mengkonfigurasi rute jaringan secara sesuai. Route Controller membutuhkan akses untuk operasi `get` terhadap objek-objek Node.
+
+v1/Node:
+
+* Get
+
+### Akses untuk Service Controller
+
+Service Controller memantau terjadinya operasi `create`, `update` dan `delete`, kemudian mengkonfigurasi Endpoint untuk Service-service tersebut secara sesuai.
+
+Untuk mengakses Service-service, Service Controller membutuhkan akses untuk operasi _list_ dan _watch_. Untuk memperbarui Service-service, dibutuhkan akses untuk operasi `patch` dan `update`.
+
+Untuk menyiapkan Endpoint bagi untuk Service-service, dibutuhkan akses untuk operasi `create`, `list`, `get`, `watch`, dan `update`.
+
+v1/Service:
+
+* List
+* Get
+* Watch
+* Patch
+* Update
+
+### Akses Lainnya
+
+Implementasi dari inti CCM membutuhkan akses untuk membuat Event, dan untuk memastikan operasi yang aman, dibutuhkan akses untuk membuat ServicAccount.
+The implementation of the core of CCM requires access to create events, and to ensure secure operation, it requires access to create ServiceAccounts.
```

Comment:
forgot to throw english sentence
```suggestion

```

Reply:
Resolved in https://github.com/kubernetes/website/pull/15335/commits/6bc040f39df16dd106905e21ec9f048d262bd41c